### PR TITLE
nodejs-week1

### PR DESCRIPTION
**Pull request by Merhawi-gebreslasie**

### CODE REVIEW

File: nodejs/week1/homework/src/backend/app.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    maxNumberOfGuests: meal.maxNumberOfGuests,
+    title: meal.title,
+    description: meal.description,
+    time: 12 + meal.id,
+    address: "asd " + meal.id,
+    price: meal.price,
+    reviews: reviews.filter((review) => review.mealId === meal.id),
+    createdAt: meal.createdAt,
+  };
+});
```
Comment (dsamourkasidis) [inline nodejs/week1/homework/src/backend/app.js]: Very nice, that way you avoid writing the same code in each route :1st_place_medal:  
what's with the "asd" address though :) :)

File: nodejs/week1/homework/src/backend/app.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+console.log(mealsWithReviews);
+
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
```
Comment (dsamourkasidis) [inline nodejs/week1/homework/src/backend/app.js]: In this case both .send and .json work but it's preferred to use .json -> https://hackyourfuture-cph.slack.com/archives/C02EXR39E5R/p1645710223738659?thread_ts=1645709966.699389&cid=C02EXR39E5R

**Review by dsamourkasidis — COMMENTED** [review]
Awesome work :+1: 
Everything works perfectly!

File: nodejs/week1/homework/src/backend/app.js (hunk 1 above)
Comment (Merhawi-gebreslasie) [reply nodejs/week1/homework/src/backend/app.js]: :) :) Yes, it doesn't make sense I put  a meaningful address now...

File: nodejs/week1/homework/src/backend/app.js (hunk 2 above)
Comment (Merhawi-gebreslasie) [reply nodejs/week1/homework/src/backend/app.js]: That was interesting ,thanks 💙.I have just implemented it .

**Pull request merged**